core/templates/domain/skill/SkillDifficultyObjectFactorySpec.ts

```diff
@@ -45,5 +45,25 @@ describe('Skill Difficulty object factory', () => {
       };
       expect(skillDifficulty.toBackendDict()).toEqual(skillDifficultyDict);
     });
+
+    it('should set the description of the skill difficulty instance', () => {
+      var skillDifficulty = skillDifficultyObjectFactory.create(
+        '1', 'test skill', 0.3);
+      expect(skillDifficulty.getId()).toEqual('1');
+      expect(skillDifficulty.getDescription()).toEqual('test skill');
+      expect(skillDifficulty.getDifficulty()).toEqual(0.3);
+      skillDifficulty.setDescription('new test skill');
+      expect(skillDifficulty.getDescription()).toEqual('new test skill');
+    });
+
+    it('should set the difficulty of the skill difficulty instance', () => {
+      var skillDifficulty =
+        skillDifficultyObjectFactory.create('1', 'test skill', 0.3);
+      expect(skillDifficulty.getId()).toEqual('1');
+      expect(skillDifficulty.getDescription()).toEqual('test skill');
+      expect(skillDifficulty.getDifficulty()).toEqual(0.3);
+      skillDifficulty.setDifficulty(0.5);
```

Comment:
Add a newline above and below (to highlight difference sections i.e, setup, action & end verification)

Reply:
Done.

Reply:
Sorry for the misunderstanding, I was asking for this:
```
    it('should set the difficulty of the skill difficulty instance', () => {
      var skillDifficulty = skillDifficultyObjectFactory.create(
        '1', 'test skill', 0.3);

      expect(skillDifficulty.getId()).toEqual('1');
      expect(skillDifficulty.getDescription()).toEqual('test skill');
      expect(skillDifficulty.getDifficulty()).toEqual(0.3);

      skillDifficulty.setDifficulty(0.5);

      expect(skillDifficulty.getDifficulty()).toEqual(0.5);
    });
```

[Make sure to undo the changes you have made in the last commit.]

Reply:
> Sorry for the misunderstanding, I was asking for this:
> 
> ```
>     it('should set the difficulty of the skill difficulty instance', () => {
>       var skillDifficulty = skillDifficultyObjectFactory.create(
>         '1', 'test skill', 0.3);
> 
>       expect(skillDifficulty.getId()).toEqual('1');
>       expect(skillDifficulty.getDescription()).toEqual('test skill');
>       expect(skillDifficulty.getDifficulty()).toEqual(0.3);
> 
>       skillDifficulty.setDifficulty(0.5);
> 
>       expect(skillDifficulty.getDifficulty()).toEqual(0.5);
>     });
> ```
> 
> [Make sure to undo the changes you have made in the last commit.]

Done. Can you look at it? Thanks!